mne/epochs.py

```diff
@@ -1566,7 +1593,7 @@ def _getitem(self, item, reason='IGNORED', copy=True, drop_event_id=True,
             return epochs
 
     def crop(self, tmin=None, tmax=None):
-        """Crop a time interval from epochs object.
+        """Crop a time interval from the epochs, which is modified in-place.
```

Comment:
All methods are/should be designed to operate in-place, I'd rather not add this to just some (or even to all) of them. If you think it's an important call-out then a separate line below would be better.